test/integration/host_js_test.rb

```diff
@@ -110,7 +110,7 @@ class HostJSTest < IntegrationTestWithJavascript
     test "all audit redirect to audit page" do
       visit host_details_page_path(@host)
       find('a', :text => /All audits/).click
-      assert_current_path audits_path
+      assert_current_path audits_path(search: "host%3D#{@host.fqdn}")
```

Comment:
looks like this test is failing, it seems that the path helper is escaping the `%3D` again
```suggestion
      assert_current_path audits_path(search: "host=#{@host.fqdn}")
```